### ISSUE TITLE
Whipper is seemling unable to interact with the CD drive on ubuntu 18.04

### ISSUE THREAD

**Issue by jimkoen** [whipper-team/whipper#483]
When I run "whipper drive analyze", the program encounters a critical error and is unable to perform. This happens on basicially every interaction with the drive, regardless of CD inserted or not.

```
user@ubuntu:~$ whipper1 drive list
drive: /dev/cdrom, vendor: ASUS    , model: DRW-24D5MT      , release: 2.00
WARNING:whipper.command.drive:no read offset found. Run 'whipper offset find'
WARNING:whipper.command.drive:unknown whether audio cache can be defeated. Run 'whipper drive analyze'
user@ubuntu:~$ whipper1 drive analyze
CRITICAL:whipper.command.main:exception TypeError at /usr/local/lib/python3.7/dist-packages/whipper-0.9.0-py3.7-linux-x86_64.egg/whipper/program/cdparanoia.py:606: failed(): sequence item 0: expected str instance, bytes found
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/whipper-0.9.0-py3.7-linux-x86_64.egg/whipper/common/task.py", line 89, in _read
    self._done()
  File "/usr/local/lib/python3.7/dist-packages/whipper-0.9.0-py3.7-linux-x86_64.egg/whipper/common/task.py", line 107, in _done
    self.failed()
  File "/usr/local/lib/python3.7/dist-packages/whipper-0.9.0-py3.7-linux-x86_64.egg/whipper/program/cdparanoia.py", line 606, in failed
    output = "".join(self._output)
TypeError: sequence item 0: expected str instance, bytes found
```

However, cdparanoia tests the drive okay:

```
cdparanoia III release 10.2 (September 11, 2008)

Using cdda library version: 10.2
Using paranoia library version: 10.2
Checking /dev/sr0 for cdrom...
	Testing /dev/sr0 for SCSI/MMC interface
		SG_IO device: /dev/sr0

CDROM model sensed sensed: ASUS DRW-24D5MT 2.00 

Checking for SCSI emulation...
	Drive is ATAPI (using SG_IO host adaptor emulation)

Checking for MMC style command set...
	Drive is MMC style
	DMA scatter/gather table entries: 1
	table entry size: 131072 bytes
	maximum theoretical transfer: 55 sectors
	Setting default read size to 27 sectors (63504 bytes).

Verifying CDDA command set...
	Expected command set reads OK.

Attempting to set cdrom to full speed... 
	drive returned OK.

=================== Checking drive cache/timing behavior ===================

Seek/read timing:
	[59:08.05]:   28ms seek, 0.37ms/sec read [36.0x]                 
	[50:00.00]:   26ms seek, 0.41ms/sec read [32.9x]                 
	[40:00.00]:   27ms seek, 0.44ms/sec read [30.1x]                 
	[30:00.00]:   33ms seek, 0.48ms/sec read [27.8x]                 
	[20:00.00]:   34ms seek, 0.55ms/sec read [24.0x]                 
	[10:00.00]:   36ms seek, 0.63ms/sec read [21.1x]                 
	[00:00.00]:   40ms seek, 0.84ms/sec read [15.8x]                 

Analyzing cache behavior...
	Approximate random access cache size: 27 sector(s)               
	Drive cache tests as contiguous                           
	Drive readahead past read cursor: 128 sector(s)                
	Cache tail cursor tied to read cursor                      
	Cache tail granularity: 0 sector(s)                      
	        Cache read speed: 0.07ms/sector [181x]
	        Access speed after backseek: 0.84ms/sector [15x]
	Backseek flushes the cache as expected

Drive tests OK with Paranoia.
```

**Comment by nathanbibb** [comment]
Not sure if this is your issue, but I had a similar problem when getting whipper to work on a new install of Fedora Server Edition.  I had to make my login user a member of the 'cdrom' group in order for my user to have access to the cdrom.  For some reason, CD Paranoia worked ok even without the user being in that group, though, if I remember correctly.

You could check logging in as root and seeing if whipper works, also.  That would point to some group/permissions issue.

**Comment by JoeLametta** [comment]
Hi, this is a duplicate of a known bug (#431) already fixed in whipper `develop` (#434) but not yet in a tagged whipper version.
